js/index.js

```diff
@@ -1354,8 +1357,10 @@ function DataManager() {
 			});
 		} else {
 			oTable.fnClearTable();
-			oTable.fnAddData(dataList);
-			oTable.fnAdjustColumnSizing();
+			if (dataList.length > 0){
+				oTable.fnAddData(dataList);
+				oTable.fnAdjustColumnSizing();
+			}
```

Comment:
Just for the record, what was the error when `dataList.length == 0` and those two methods were called?